opentracing_instrumentation/client_hooks/__init__.py

```diff
@@ -54,6 +54,28 @@ def install_all_patches():
     requests.install_patches()
 
 
+def enable_specific_span_extractor(span_extractor_func):
```

Comment:
Let's minimize the number of new names being introduced: "specific", "span extractor", "parent_span_func".

The code is currently using `get_current_span`. I suggest
  * add a new private module `client_hooks._current_span`
  * add a single global var `current_span_func` to `_current_span` module, instead of "parent_span_func" in each patching module
  * add a global function `set_current_span_func` to `_current_span` module
  * import `set_current_span_func` from `__init__`
  * all patching modules should use `_current_span.current_span_func()`

 This way we're always using the same naming convention revolving around "current span". And we don't create the possibility of mis-configuration where each patching module has its own extractor function (if there is a requirement for this flexibility in the future we can consider it then).